docs/tutorials/01_KalmanFilterTutorial.py

```diff
@@ -409,8 +309,4 @@
     ax.add_artist(ellipse)
 fig
 
-# sphinx_gallery_thumbnail_number = 4
-
-# %%
-# There are situations in which linearisation of the problem is not useful/possible.
-# For this, we require an adjusted process, as tackled by the **Extended Kalman Filter**.
+# sphinx_gallery_thumbnail_number = 4
```

Comment:
Thumbnail doesnt appear on Tutorials page.

Reply:
I haven't taken the trouble to work out what this means - again, happy for you to go ahead and make it work.